modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/BuilderScreen/BuilderScreen.tsx

```diff
@@ -51,113 +51,123 @@ export function BuilderScreen({
 
 	return (
 		<>
-			<ManagementToolbar.Container>
-				<ManagementToolbar.ItemList expand>
-					<ManagementToolbarSearch
-						query={query}
-						setQuery={setQuery}
-					/>
-
-					<ManagementToolbar.Item>
-						<ClayButtonWithIcon
-							aria-label={Liferay.Language.get('add')}
-							className="nav-btn nav-btn-monospaced"
-							onClick={openModal}
-							symbol="plus"
-						/>
-					</ManagementToolbar.Item>
-				</ManagementToolbar.ItemList>
-			</ManagementToolbar.Container>
-
 			{tableItems.length ? (
 				<ClayList>
-					{tableItems.map((viewColumn, index) => (
-						<React.Fragment key={viewColumn?.objectFieldName}>
-							{index === 0 && (
-								<ClayList.Item flex>
-									<ClayList.ItemField
-										className={classNames(
-											'lfr-object__object-builder-screen-first-column',
-											!hasDragAndDrop &&
-												'lfr-object__object-builder-screen-first-column--not-draggable'
-										)}
-										expand
-									>
-										{firstColumnHeader}
-									</ClayList.ItemField>
+					<ClayList.Item className="lfr-object__object-builder-screen-list">
```

Comment:
This is a list item, not a list, so this `className` is incorrect.

Reply:
It looks to me that this element should not exist, because there is already a `ClayList` wrapping the list items.